mxcube3/routes/SampleCentring.py

```diff
@@ -242,6 +242,36 @@ def get_shapes():
     return resp
 
 
+@mxcube.route("/mxcube/api/v0.1/sampleview/shapes/<sid>", methods=['GET'])
+def get_shape_with_sid(sid):
+    """
+    Retrieve all the stored centred positions.
```

Comment:
Check this comment, status 409 is never set if there is an error and it refers to centered position where I should be shape